src/CocktailCard/CocktailCard.scss

```diff
@@ -1,35 +1,37 @@
 .cocktail-card {
+	background-image: url('https://media-cdn.getbento.com/accounts/b407703cbc06b7de17a1aab05567665c/media/accounts/media/TBBjpsToRMaQp7vLG6Ty_border-image-white-decor.png');
+	background-repeat: no-repeat;
+	background-size: cover;
 	align-items: center;
-	border: 2px solid grey;
 	display: flex;
 	flex-direction: column;
 	font-family: 'Recursive', sans-serif;
 	justify-content: center;
 	margin: 2%;
-	max-height: 55%;
-	max-width: 25%;
-	min-height: 55%;
-	min-width: 20%;
-	opacity: 100%;
+	max-height: 290px;
+	min-height: 290px;
+	max-width: 250px;
+	min-width: 250px;
```

Comment:
background-image property, w/ a few other helper background properties. made the min & mx heights/widths tall the same so the card's can be very steady throughout screen size changes @LeahWinters @Melizzo @asodemann18